data/vibe/data/json.d

```diff
@@ -2010,7 +2016,6 @@ void writeJsonString(R, bool pretty = false)(ref R dst, in Json json, size_t lev
 			dst.put('{');
 			bool first = true;
 			foreach (string k, ref const Json e; json.byKeyValue) {
-				if( e.type == Json.Type.undefined ) continue;
```

Comment:
Seconded. This was done to be consistent with the ECMA script spec, from which the `undefined` value originates.